source/configuration_manual/push_notification.rst

```diff
@@ -537,6 +542,8 @@ Example configuration:
 
 .. code-block:: none
 
+  mail_plugins = $mail_plugins notify push_notification chronos_push_notification
+
   plugin {
     push_notification_driver = chronos:url=http://login:pass@node1.domain.tld:8009/chronos/v1/itip/pushmail max_retries=2 timeout=2500ms msg_max_size=1mb
```

Comment:
do we mention anywhere that all the stuff after url is optional?

Reply:
Yes the table above this example states required vs. optional arguments.